public/js/src/game/ui/HanabiCard.js

```diff
@@ -75,6 +75,7 @@ class HanabiCard extends graphics.Group {
         this.negativeColorClues = [];
         this.specialRankSuitRemoved = false;
         this.turnClued = null;
+        this.turnsClued = [];
```

Comment:
`learnedCards` is a separate global because rewinding rebuilds the rest of the game state, including the cards. This means that if a player rewinds before the latest turn a given card was clued and then middle-clicks it, they will not move forward.

Reply:
"This also works in the replay, allowing you to middle-click a card several times to go backwards through each time it was clued."

That's what I'm intending. Should this also be written in FEATURES.md?